src/reducers/targetReducer.js

```diff
@@ -11,6 +12,10 @@ export default function targetReducer(state = initialState, action) {
       return state.setIn(['targetList'], action.targets);
     case actions.LOAD_TARGETS_FAILED:
       return [];
+    case actions.ADD_TARGET_SUCCESS:
+      return state.setIn(['target'], action.target);
```

Comment:
If `action.target` is a plain javascript object, you will need to change it to an immutable object

Reply:
Its done!!